install.sh

```diff
@@ -1,61 +1,64 @@
 #!/bin/bash
 
-#Updating packages
-apt-get --yes update
-apt-get --yes upgrade
+# Updating packages
+apt --yes update
+apt --yes upgrade
 
-#Installing dependencies
-apt-get --yes --install-recommends install dnsmasq ldm-ubuntu-theme
-apt-get --yes --install-recommends install ltsp-server
-export DEBIAN_FRONTEND=noninteractive
-apt-get -yq --yes --install-recommends install ltsp-client
-apt-get --yes install epoptes epoptes-client
+# Installing dependencies
+apt --yes --install-recommends install dnsmasq ldm-ubuntu-theme
+apt --yes --install-recommends install ltsp-server
+DEBIAN_FRONTEND=noninteractive apt --yes --install-recommends install ltsp-client
+apt --yes install epoptes epoptes-client
 
-
-#Adding vagrant user to group epoptes
+# Adding vagrant user to group epoptes
 gpasswd -a ${SUDO_USER:-$USER} epoptes
 
-#Updating kernel
+# Updating kernel
 echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
-#configure dnsmasq
+# Configure dnsmasq
 ltsp-config dnsmasq
 
-#Creating lts.conf
+# enabling password authentication 
+sed -i /etc/ssh/sshd_config \
+    -e "/Authentication no\$/ c PasswordAuthentication yes"
+service ssh restart
+
+# Creating lts.conf
 ltsp-config lts.conf
 
-#Installing additional software
-apt-get --yes install edubuntu-desktop
-apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
+# Installing additional software
+apt --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
-#Installing ltsp-manager
-add-apt-repository ppa:ts.sch.gr
-apt-get --yes update
-apt-get --yes install ltsp-manager
+# Installing ltsp-manager
+echo | add-apt-repository ppa:ts.sch.gr
```

Comment:
"echo yes" you mean?

Reply:
it asks for enter. We can use `echo enter` as well.
But this is cleaner So I used this one.
I am attaching screenshot for output of both - https://image.ibb.co/nA4Xsy/echoaddrepo.png

Reply:
okay.